index.js

```diff
@@ -13,6 +14,7 @@ module.exports = (target, opts) => {
 	let appArgs = [];
 	let args = [];
 	const cpOpts = {};
+	const isWSL = (os.type() === 'Linux' && os.release().indexOf('Microsoft') > -1);
```

Comment:
Yes, I'm on the latest updates. I just did all the updates yesterday. But you're right it didn't install the latest Ubuntu:

```
sindresorhus@SINDRESORHU8D24:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.5 LTS
Release:        14.04
Codename:       trusty
```

I've added a `os.release()` check too for when it's available: https://github.com/sindresorhus/is-wsl/blob/60ea5d57a51ee596cb144ef47187c0476a5a421b/index.js#L10

Can you change to use this module?